ember-client/src/main/scala/org/http4s/ember/client/EmberClientBuilder.scala

```diff
@@ -164,9 +138,22 @@ final class EmberClientBuilder[F[_]: Concurrent: Timer: ContextShift] private (
           .withOnReaperException(_ => Applicative[F].unit)
       pool <- builder.build
     } yield {
-      val client = Client[F](request =>
+      val client = Client[F] { request =>
+        def getValidManaged: Resource[F, Managed[F, (RequestKeySocket[F], F[Unit])]] =
+          pool.take(RequestKey.fromRequest(request)).flatMap { managed =>
+            Resource
+              .liftF(managed.value._1.socket.isOpen)
```

Comment:
Clients were getting dead connections out of the pool. This doesn't fix, but is at least a little help.